include/aws/common/error_codes.h

```diff
@@ -0,0 +1,63 @@
+#ifndef AWS_COMMON_ERROR_CODES_H
+#define AWS_COMMON_ERROR_CODES_H
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+enum aws_common_error {
```

Comment:
Maybe we could merge error_codes.h and error.h?

Reply:
Done